docs/team/TSAI-HSIAO-HAN.adoc

```diff
@@ -62,7 +62,7 @@ Even if users do not close the notification, the notification will disappear aft
 gives me the idea of how to implement a reminder notification.
 
 * *Code contributed*: +
-Please click these links to see a sample of my code.[https://nus-cs2103-ay1920s1.github.io/tp-dashboard/#search=tsai-hsiao-han&sort=groupTitle&sortWithin=title&since=2019-09-06&timeframe=commit&mergegroup=false&groupSelect=groupByRepos&breakdown=false[Code]]
+Please click these links to see a sample of my code.[https://nus-cs2103-ay1920s1.github.io/tp-dashboard/#search=tsai-hsiao-han&sort=groupTitle&sortWithin=title&since=2019-09-06&timeframe=commit&mergegroup=false&groupSelect=groupByRepos&breakdown=false[Here]]
```

Comment:
```suggestion
Please click this link to see a sample of my code. [https://nus-cs2103-ay1920s1.github.io/tp-dashboard/#search=tsai-hsiao-han&sort=groupTitle&sortWithin=title&since=2019-09-06&timeframe=commit&mergegroup=false&groupSelect=groupByRepos&breakdown=false[Here]]
```